filters/ratelimit/ratelimit.go

```diff
@@ -129,8 +131,8 @@ func NewRatelimit(provider RatelimitProvider) filters.Spec {
 //    backendHealthcheck: Path("/healthcheck")
 //    -> clusterRatelimit("groupA", 200, "1m", 503)
 //    -> "https://foo.backend.net";
-func NewClusterRateLimit(provider RatelimitProvider) filters.Spec {
-	return &spec{typ: ratelimit.ClusterServiceRatelimit, provider: provider, filterName: filters.ClusterRatelimitName}
+func NewClusterRateLimit(provider RatelimitProvider, maxKeyShards int) filters.Spec {
```

Comment:
We could also export a new func: `NewClusterRateLimitWithMaxKeyShards(provider RatelimitProvider, maxKeyShards int)` and let `NewClusterRateLimit(provider RatelimitProvider){ return NewClusterRateLimitWithMaxKeyShards(provider, 1)}`

Reply:
Fixed